topic_message_query_e2e_test.go

```diff
@@ -323,3 +323,31 @@ func TestIntegrationTopicMessageQueryNoStartTime(t *testing.T) {
 	err = CloseIntegrationTestEnv(env, nil)
 	require.NoError(t, err)
 }
+
+func TestIntegrationTopicMessageQueryCanExecuteWithTls(t *testing.T) {
+	client := ClientForNetwork(map[string]AccountID{})
+	client.SetMirrorNetwork([]string{"mainnet-public.mirrornode.hedera.com:443"})
```

Comment:
no `client.SetTransportSecurity(true)`?

Reply:
nvm. just saw ManagedNode sets TLS accordingly depending on the port number.

For the test case failure, you can keep the topic id and set limit to 10. there is dynamic performance issue.

Reply:
TYSM for the limit 10 recommendation. That did the trick, but definitely a weird issue for limit of 1 to not work. Anyways I force set `SetTransportSecurity(true)` just in case.